app/code/Magento/Ui/Test/Mftf/ActionGroup/AdminDataGridFilterActionGroup.xml

```diff
@@ -1,44 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@lbajsarowicz Could you please explain why this file can be deleted?

Reply:
Because...